precompile/testutils/test_precompile.go

```diff
@@ -0,0 +1,86 @@
+// (c) 2023, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+
+package testutils
```

Comment:
I can put this back into `contract` to avoid a new package if others prefer.

Reply:
Good idea